src/Compilers/Core/Portable/Syntax/SyntaxList`1.cs

```diff
@@ -11,7 +11,7 @@
 namespace Microsoft.CodeAnalysis
 {
     /// <summary>
-    /// A list of <see cref="SyntaxNode"/>.
+    /// A list of <see cref="SyntaxNode"/>. 
```

Comment:
This file is still modified!

---
In reply to: [65650591](https://github.com/dotnet/roslyn/pull/11681#discussion_r65650591) [](ancestors = 65650591)